--- libtool-2.4.6/tests/mdemo.at.orig	2015-12-24 17:04:56.988843856 -0500
+++ libtool-2.4.6/tests/mdemo.at	2015-12-24 17:05:12.458857065 -0500
@@ -700,7 +700,7 @@ LT_AT_MAKE([main.$objext])
 
 # Making object files
 # ls -l in MSYS sometimes shows year, not time, for really fresh files.
-sleep 1
+sleep 3
 ls -l . "$objdir" | $EGREP -v '(^total|testsuite.log$)' > "$before"
 force_dry_run=:
 LT_AT_MAKE([foo1.lo foo2.lo libsub.la])
@@ -712,7 +712,7 @@ force_dry_run=false
 LT_AT_MAKE([foo1.lo foo2.lo libsub.la])
 
 # Making libraries
-sleep 1 # for MSYS
+sleep 3 # for MSYS
 ls -l . "$objdir" | $EGREP -v '(^total|testsuite.log$)' > "$before"
 force_dry_run=:
 LT_AT_MAKE([foo1.la libfoo2.la])
@@ -724,7 +724,7 @@ force_dry_run=false
 LT_AT_MAKE([foo1.la libfoo2.la])
 
 # Making programs
-sleep 1 # for MSYS
+sleep 3 # for MSYS
 ls -l . "$objdir" | $EGREP -v '(^total|testsuite.log$)' > "$before"
 force_dry_run=:
 LT_AT_MAKE([mdemo$EXEEXT mdemo_static$EXEEXT])
@@ -737,7 +737,7 @@ $lt_INSTALL -d "$prefix/bin"
 $lt_INSTALL -d "$prefix/include"
 $lt_INSTALL -d "$prefix/lib"
 
-sleep 1 # for MSYS
+sleep 3 # for MSYS
 ls -l . "$objdir" | $EGREP -v '(^total|testsuite.log$)' > "$before"
 ls -lR "$prefix" | $EGREP -v '(^total|testsuite.log$)' >> "$before"
 force_dry_run=:
@@ -755,7 +755,7 @@ LT_AT_MAKE([install])
 # Libtool does not uninstall the programs, remove them first
 rm -f "$prefix/bin/mdemo$EXEEXT" "$prefix/bin/mdemo_static$EXEEXT"
 
-sleep 1 # for MSYS
+sleep 3 # for MSYS
 ls -l . "$objdir" | $EGREP -v '(^total|testsuite.log$)' > "$before"
 ls -lR "$prefix" | $EGREP -v '(^total|testsuite.log$)' >> "$before"
 force_dry_run=:
